beringei/tools/plain_text_service/DateUtils.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#pragma once
+
+#include <ctime>
+
+namespace facebook {
+namespace gorilla {
+
+class DateUtils {
+ public:
+  // Helper function to parse iso timestamps
+  // For example, "2016-11-17T02:12:40.708Z".
+  static bool stringParseIsoTimestamp(const std::string& str, time_t* t) {
+    const char* format = "%Y-%m-%dT%H:%M:%S";
+    struct tm timeinfo;
+    memset(&timeinfo, 0, sizeof(struct tm));
+    if (strptime(str.c_str(), format, &timeinfo) == nullptr) {
+      return false;
+    }
+
+    timeinfo.tm_isdst = -1; // no DST information available
```

Comment:
Please rewrite the comments to be full sentences with capitalization/punctuation throughout, e.g., "No DST information available."